tests/foreman/api/test_smartproxy.py

```diff
@@ -23,7 +23,6 @@
 from robottelo.api.utils import one_to_many_names
 
 
-@skip_if_bug_open('bugzilla', 1262037)
```

Comment:
This is not the right way to skip an entire class. This actually messes up with the class and return a function instead.